### ISSUE TITLE
Daylight Saving Time is not handled

### ISSUE THREAD

**Issue by ghost** [saxbophone/solar-time#1]
By a pure chance i've landed on your Solar Time web page and found that the DST is not handled, so it returns an invalid time.

https://en.wikipedia.org/wiki/Daylight_saving_time

**Comment by saxbophone** [comment]
Hi, thanks for noticing :smile: – Yes that is true, I realised this soon after making the app and figured that it makes sense for it to be an hour out (when I developed this my timezone was BST). This is on the basis that Daylight savings is a human concept primarily used as a convenience factor for dark seasons of the year, whereas the app was mainly meant to reflect the time based on the exact distance from 0º Longitude, where that point is in UTC.

Having said all this, it would be nice to have a feature to disable/enable the resolution of Daylight Savings time via a checkbox of some sort, depending on use case. The primary use case I had when making this was to see how many minutes behind/ahead I was at any given location. What would be the main advantage of handling DST from your point of view?
